src/cordova/storage.ts

```diff
@@ -106,7 +107,8 @@ async function prepareStorage(secureStorage: CordovaSecureStorage) {
   await Promise.all([
     addPlaceholderKey(storeKeys.keystore, {}),
     addPlaceholderKey(storeKeys.settings, {}),
-    addPlaceholderKey(storeKeys.ignoredSignatureRequests, [])
+    addPlaceholderKey(storeKeys.ignoredSignatureRequests, []),
+    addPlaceholderKey(storeKeys.clientSecret, "TODO_RANDOM_NUMBER")
```

Comment:
We still have to add the generation of the random number here but I wasn't sure how to do this best.